app/src/main/java/org/simple/clinic/AppDatabase.kt

```diff
@@ -134,7 +134,7 @@ import org.simple.clinic.drugs.search.Answer as DrugAnswer
     DrugFrequency.RoomTypeConverter::class,
     Outcome.RoomTypeConverter::class,
     QuestionnaireType.RoomTypeConverter::class,
-    QuestionnaireLayout.RoomTypeConverter::class
+    BaseComponentData.RoomTypeConverter::class
```

Comment:
Would `QuestionnaireBaseComponent` be a better name?

Reply:
@siddh1004 this change isn't necessary, because `BaseComponent`'s parent namespace already contains `Questionnaire`